code/mob/living/critter/spider.dm

```diff
@@ -459,3 +590,79 @@
 				I.throw_at(target, 12, 3)
 
 	return bloods
+
+/* ====================================================== */
+/* --------------------- AI Spiders --------------------- */
+/* ====================================================== */
```

Comment:
i don't get why we need this to be a subtype at all

Reply:
mostly because I didn't want to change the behaviour of existing mobs that might be used for other things, but also because having a subtype that has AI means you can spawn player controlled spider mobs easily, and also AI controlled ones.

Reply:
Why not just have the AI be active only if there's no client

Reply:
I can totally do that if preferred, it just would have altered the existing mob behaviour. Also for consistency with like, monkey and monkey/npc

Reply:
I think it'd be the best and most clean to not have it be a part of the path.

Reply:
done, spiders are now AI unless they have a client.